org.thonny.Thonny.yaml

```diff
@@ -0,0 +1,314 @@
+---
+id: org.thonny.Thonny
+branch: stable
+runtime: org.freedesktop.Sdk
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+command: thonny
+cleanup:
+  - /lib/debug
+  - /man
+  - /share/man
+finish-args:
+  # Thonny requires access to serial / USB devices for embedded development.
+  - --device=all
+
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+modules:
+  # The `tkinter` module is missing from the Freedesktop Sdk's Python installation.
+  - name: tkinter
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=${FLATPAK_DEST} .
+    sources:
+      - type: git
+        url: https://github.com/iwalton3/tkinter-standalone
+        commit: ba946536054f9d27a08aafde21aa18330ce05729
+    modules:
+      - name: tcl
+        buildsystem: autotools
+        subdir: unix
+        build-options:
+          no-debuginfo: true
+        cleanup:
+          - /bin
+          - /lib/debug
+          - /lib/pkgconfig
+          - /man
+        sources:
+          - type: archive
+            url: https://prdownloads.sourceforge.net/tcl/tcl8.6.11-src.tar.gz
+            sha256: 8c0486668586672c5693d7d95817cb05a18c5ecca2f40e2836b9578064088258
+      - name: tk
+        buildsystem: autotools
+        subdir: unix
+        build-options:
+          no-debuginfo: true
```

Comment:
```suggestion
```
Is this necessary? It's better to have the debug info, they are extracted in a separate package.

Reply:
Nope, it's not necessary. I'll remove that option.